src/java/org/apache/cassandra/db/lifecycle/LogRecord.java

```diff
@@ -148,16 +148,16 @@ public static LogRecord makeAbort(long updateTime)
 
     public static LogRecord make(Type type, SSTable table)
     {
-        String absoluteTablePath = absolutePath(table.descriptor.baseFilename());
-        return make(type, getExistingFiles(absoluteTablePath), table.getAllFilePaths().size(), absoluteTablePath);
+        String absoluteTablePath = table.descriptor.baseFileUri() + Component.separator;
```

Comment:
q: why the file system specific separator is taken from `Component`?

Reply:
it's not file system specific separator,  it's the `-` inside file name